src/components/dashboard/users/UserPasswordForm.tsx

```diff
@@ -7,13 +7,15 @@ import loading from '../../loading/loading';
 import toast from '../../toast/toast';
 import ButtonElement from '../../../elements/ButtonElement';
 import InputElement from '../../../elements/InputElement';
+import { UserDto } from '@jellyfin/sdk/lib/generated-client';
```

Comment:
```suggestion
import type { UserDto } from '@jellyfin/sdk/lib/generated-client';
```

This should be moved to the top of the list of imports also.